pandas/tests/tools/test_to_timedelta.py

```diff
@@ -197,7 +197,8 @@ def test_to_timedelta_on_missing_values(self):
         )
         tm.assert_series_equal(actual, expected)
 
-        actual = to_timedelta(Series(["00:00:01", pd.NaT]))
+        with tm.assert_produces_warning(FutureWarning, match="Inferring timedelta64"):
```

Comment:
can you have *some* of the tests also assert NO warning if the data is not wrapped in a Series, e.g. just passed to `to_timedelta` or `to_datetime` for example should not warn and have the same result as the below.

Reply:
updated + green

Reply:
might be worth doing for some calls to to_datetime as well (followon ok)